docs/blog/20240918/20240918.md

```diff
@@ -0,0 +1,73 @@
+# L25GC+: Revolutionizing the Control Plane for 5G Core Networks
+>[!NOTE]
+> Author: Chun-Ting Lin
+> Date: 2024/09/18
+---
+
+## Introduction
+With the rapid development of 5G technology, the demand for network performance is also increasing. Conventional 5G core networks handle control plane operations with high latency due to the communication overhead of service-based interfaces (SBIs), which cannot meet the demands of low-latency applications such as the Internet of Things (IoT) and the automotive Internet of Things (IoT). To this end, the researchers propose L25GC+, a shared memory-based control plane design for 5G core networks, which aims to significantly reduce latency and improve system performance.
+
+## Background of L25GC+
+L25GC+ is a 5G control plane design that utilizes shared memory processing to reduce control plane delay.
+
+## Technical Principle of L25GC+
+- **Shared memory I/O stack** 
+    - Shared memory processing with lock-free rings
+    - Like L25GC, built with OpenNetVM
```

Comment:
I suggest you to remove "Like L25GC".
It's because of the target audiences may not heard it before.